app/styles/app.sass

```diff
@@ -1345,3 +1345,48 @@ input:checked + .slider:before
 
 .slider.round:before
   border-radius: 50%
+
+// attachments
+
+$attachment-icon-width: 4em
+.attachment
+  position: relative
+  display: flex
+  align-items: center
+  background: #f5f5f5
+  color: $text
+  margin-bottom: 1em
+  &:hover, &:focus, &:active
+    color: $primary
+  &::before
+    content: attr(data-type)
+    text-transform: uppercase
+    letter-spacing: 0.05em
+    font-size: 0.8em
+    font-weight: bold
+    display: block
+    width: $attachment-icon-width
+    height: $attachment-icon-width / 1.4
+    text-align: center
+    background: $grey
+    color: white
+    display: flex
+    align-items: center
+    justify-content: center
+  &__label
+    max-width: 35em
+    display: inline-block
+    overflow: hidden
+    text-overflow: ellipsis
+    white-space: nowrap
+  &__icon
+    position: absolute
+    right: 1em
+
+$file-extension-list:   log     apk     ipa     pdf     doc     txt     zip     jpg     jpeg    png     bmp
```

Comment:
@Praseetha-KR can you add `xls`, `xlsx` and `xls` ?

Reply:
or you can have a default colour for extension not matching

Reply:
@subho007 Added spreadsheet extensions in 73915ca. Default color is already there:
https://github.com/appknox/irene/blob/73915ca1b7ea9e6771db5890cb330b6cf89b3fa7/app/styles/app.sass#L1370
Also modified extension color loop to use sass map instead of lists. (Since multiline map is not supported by `sass`, moved it to an `scss` file & imported.)